### PR TITLE
Fixed the issue on update membership

### PR DESCRIPTION
**Pull request by mkbeefcake**
Fixed https://github.com/Joystream/pioneer/issues/3133

https://user-images.githubusercontent.com/69617937/206702505-3ed71081-8aad-4dbc-bf6a-df80e9d3cbf4.MP4

### CODE REVIEW

File: packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx [hunk 1]
```diff
@@ -46,9 +46,8 @@ export const UpdateMembershipModal = () => {
     )
   }
 
-  if (state.matches('success')) {
-    setActive(member)
-    return <UpdateMembershipSuccessModal onClose={hideModal} member={member} />
+  if (state.matches('success') && active != undefined) {
```
Comment (oleksanderkorn) [inline packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx:49]: Maybe `&& active` or `&& active !== undefined` is better?

**Review by oleksanderkorn — APPROVED** [review]

File: packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx (hunk 1 above)
Comment (traumschule) [reply packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx:49]:
```suggestion
  if (active && state.matches('success')) {
```

Comment (mkbeefcake) [reply packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx]: I have fixed the problem on the Jest test code and changed the code as following @traumschule.

**Review by traumschule — APPROVED** [review]

File: packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx [hunk 2]
```diff
@@ -46,9 +46,8 @@ export const UpdateMembershipModal = () => {
     )
   }
 
-  if (state.matches('success')) {
-    setActive(member)
-    return <UpdateMembershipSuccessModal onClose={hideModal} member={member} />
+  if (active && state.matches('success')) {
+    return <UpdateMembershipSuccessModal onClose={hideModal} member={active} />
```
Comment (thesan) [inline packages/ui/src/memberships/modals/UpdateMembershipModal/UpdateMembershipModal.tsx:50]: :+1:

**Review by thesan — APPROVED** [review]

**Pull request merged**